homeassistant/components/twitter/notify.py

```diff
@@ -80,14 +82,17 @@ def send_message(self, message="", **kwargs):
                 _LOGGER.warning("'%s' is not a whitelisted directory", media)
                 return
 
-        callback = partial(self.send_message_callback, message)
+        if target:
+            callback = partial(self.send_message_callback, message, target)
+        else:
+            callback = partial(self.send_message_callback, message, self.default_user)
```

Comment:
Instead of duplicating the partial, creating a local user variable passing in the partial will work too (making the code a little more slick)